### ISSUE TITLE
When a list of formats is given, only the first format is used

### ISSUE THREAD

**Issue by PeterDraex** [xulihang/barcode-detector-polyfill#2]
I am trying the library and found a bug. When the BarcodeDetector is configured to receive two barcode formats, only the first format is being detected.

```
barcodeDetector = new window.BarcodeDetector({ formats: ["data_matrix", "qr_code"] }); // decodes only data_matrix (should decode both)
barcodeDetector = new window.BarcodeDetector({ formats: ["qr_code", "data_matrix"] }); // decodes only qr_code (should decode both)
barcodeDetector = new window.BarcodeDetector(); // decodes both data_matrix and qr_code (correct behavior)
```

I've tested it with the example code from [README.md](https://github.com/xulihang/barcode-detector-polyfill/blob/b38551b7b10c6bc2c7d8055c608a0873f0cf4f2d/README.md).

**Comment by xulihang** [comment]
Are you using ZXing? It can only read one code at a time.

**Comment by xulihang** [comment]
I can reproduce this problem. Will try to fix it.

**Comment by xulihang** [comment]
Fixed with this commit: https://github.com/xulihang/barcode-detector-polyfill/commit/ac9b69b522d43d2f656fe186ed8cfa5a9fd452e4

I've also released a new version 0.6.5 for this.
